go/test/endtoend/vtgate/schematracker/restarttablet/schema_restart_test.go

```diff
@@ -86,7 +90,10 @@ func TestMain(m *testing.M) {
 
 		// restart the tablet so that the schema.Engine gets a chance to start with existing schema
 		tablet := clusterInstance.Keyspaces[0].Shards[0].PrimaryTablet()
-		tablet.VttabletProcess.ExtraArgs = []string{"-queryserver-config-schema-change-signal"}
+		tablet.VttabletProcess.ExtraArgs = []string{
+			"-queryserver-config-schema-change-signal",
+			fmt.Sprintf("-queryserver-config-schema-change-signal-interval=%d", signalInterval),
```

Comment:
nit: easier way to give this is
`[]string{...., "-queryserver-config-schema-change-signal-interval", strconv.Itoa(signalInterval), .... }`